sdks/router-sdk/src/constants.ts

```diff
@@ -8,8 +8,18 @@ export const ADDRESS_THIS = '0x0000000000000000000000000000000000000002'
 export const ZERO = JSBI.BigInt(0)
 export const ONE = JSBI.BigInt(1)
 
-// = 1 << 23 or 100000000000000000000000
-export const V2_FEE_PATH_PLACEHOLDER = 8388608
+// = 1 << 23 or 0x0100000000000000000000000
```

Comment:
nit: since 1 << 23 is the actual can the comment just be:
// 0b0100000000000000000000000

you have 0x but I believe it's binary, so 0b.